examples/blog/config.js

```diff
@@ -14,6 +14,14 @@
             return {configuration: null};
         },
         componentDidMount: function () {
+            var restful = this.refs.admin.state.restful;
```

Comment:
Does [this way](https://github.com/marmelab/react-admin/commit/325d3e5deb6e5499f574dd4b4ff8f6d08ad00856?w=1) sounds better ?